### PR TITLE
SharedComputeValue: functions called only once also with later transformations

### PR DESCRIPTION
**Pull request by markotoplak**

### CODE REVIEW

**Comment by markotoplak** [comment]
@nikicc @astaric

**Comment by markotoplak** [comment]
@nikicc @astaric This should work now.

**Comment by nikicc** [comment]
It works on my data. Thanks!

File: Orange/data/table.py [hunk 1]
```diff
@@ -35,6 +35,12 @@ def get_sample_datasets_dir():
 dataset_dirs = ['', get_sample_datasets_dir()]
 
 
+"""Domain conversion cache used in from_table. It is global so that
```
Comment (astaric) [inline Orange/data/table.py]: from_table -> Table.from_table (although it is the only such method, I would be explicit)

**Comment by astaric** [comment]
I have restarted the Travis build since it failed on an unrelated error.

The rest looks ok to me. Is this still considered to be WIP?

**Comment by markotoplak** [comment]
@astaric I think it is ready for merging.

**Pull request merged**